testing/test_config.py

```diff
@@ -352,6 +352,24 @@ def test_inifilename(self, tmpdir):
         assert config.inicfg.get('name') == 'value'
         assert config.inicfg.get('should_not_be_set') is None
 
+    def test_consider_plugin(self, testdir):
```

Comment:
Also, when creating commit messages you can add a line "Fix #1618" so it closes the issue automatically when merged.